### PR TITLE
Upgrade Node.js to new LTS

### PR DESCRIPTION
**Pull request by cyranix**
See [EN-11118](https://socrata.atlassian.net/browse/EN-11118).

Also... 
* organizes the README by language instead of alphabetically
* removes yet another unused image
* removes a copypasta file

### CODE REVIEW

File: nodejs/Dockerfile [hunk 1]
```diff
@@ -2,8 +2,7 @@ FROM socrata/base
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get -y install nodejs npm
 RUN npm install -g n
-RUN n latest
```
Comment (cyranix) [inline nodejs/Dockerfile]: This is potentially dangerous when crossing major version boundaries.

File: nodejs/Dockerfile [hunk 2]
```diff
@@ -2,8 +2,7 @@ FROM socrata/base
...
-RUN n io latest
```
Comment (cyranix) [inline nodejs/Dockerfile]: `io.js` is a dead project.

File: runit-nodejs/Dockerfile [hunk 3]
```diff
@@ -5,7 +5,7 @@ RUN apt-get -y update && apt-get install -y libyajl2
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get -y install nodejs npm
 RUN npm install -g n
-RUN n 5.1
+RUN n 6.9.1
```
Comment (cyranix) [inline runit-nodejs/Dockerfile]: The 5.x series is not part of the LTS schedule, but 6.x is.

File: runit-java8-python/Dockerfile [hunk 4]
```diff
@@ -1,9 +0,0 @@
-FROM socrata/runit-java8
```
Comment (jhaynes) [inline runit-java8-python/Dockerfile]: Why get rid of this?

File: runit-nodejs/README.md [hunk 5]
```diff
@@ -1,56 +0,0 @@
-socrata/runit
```
Comment (jhaynes) [inline runit-nodejs/README.md]: This info should land somewhere. The runit containers are substantially different than the way socrata/base works.

File: README.md [hunk 6]
```diff
@@ -1,9 +1,8 @@
-shipyard
-========
+# shipyard
```
Comment (jhaynes) [inline README.md]: 👍

File: runit-nodejs/README.md (hunk 5 above)
Comment (cyranix) [inline runit-nodejs/README.md]: This info is in the `runit` container's README. This file is pure copypasta and doesn't say anything about `runit-nodejs` specifically.

File: runit-java8-python/Dockerfile (hunk 4 above)
Comment (cyranix) [inline runit-java8-python/Dockerfile]: The image is no longer in use. Confirmed with Admin team.

**Pull request merged**